.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+	"presets": [
```

Comment:
Just needs spaces instead of tabs for indent in this file.

Reply:
Ok, my editor betray me